docs/siem/timeline/api/timeline-api-create.asciidoc

```diff
@@ -0,0 +1,462 @@
+      {
+        "id": "event.category"
+      },
+      {
+        "id": "event.action"
+      },
+      {
+        "id": "host.name"
+      }
+    ],
+    "dataProviders": [ <1>
+      {
+        "and": [
+          {
+            "name": "event.category",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "event.category",
+              "value": "process",
+              "operator": ":"
+            }
+          },
+          {
+            "name": "user.name",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "user.name",
+              "value": "SYSTEM",
+              "operator": ":"
+            }
+          }
+        ],
+        "enabled": true,
+        "excluded": false,
+        "name": "host.os.platform",
+        "queryMatch": {
+          "field": "host.os.platform",
+          "value": "windows",
+          "operator": ":"
+        }
+      }
+    ],
+    "dateRange": {
+      "end": 1594005719000,
+      "start": 1593832919000
+    },
+    "description": "Gets Windows system processes from all hosts",
+    "title": "Windows system processes"
+  }
+}
+--------------------------------------------------
+// KIBANA
+
+<1> To ensure the timeline is displayed correctly in the UI, specify these
+fields in all `dataProviders` objects:
+
+* `and` (can be empty)
+* `name`
+* `enabled`
+* `excluded`
+* `queryMatch`
+** `field`
+** `value`
+** `operator`
+
+*Example 2*
+
+Creates a new timeline template:
+
+[source,console]
+--------------------------------------------------
+POST api/timeline
+{
+  "timeline": {
+    "columns": [
+      {
+        "id": "@timestamp"
+      },
+      {
+        "id": "user.name"
+      },
+      {
+        "id": "event.category"
+      },
+      {
+        "id": "event.action"
+      },
+      {
+        "id": "host.name"
+      }
+    ],
+    "dataProviders": [
+      {
+        "and": [
+          {
+            "name": "event.category",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "event.category",
+              "operator": ":",
+              "value": "process" <1>
+            }
+          },
+          {
+            "name": "user.name",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "user.name",
+              "operator": ":",
+              "value": "SYSTEM"
+            }
+          }
+        ],
+        "enabled": true,
+        "excluded": false,
+        "name": "host.os.platform",
+        "queryMatch": {
+          "field": "host.os.platform",
+          "operator": ":",
+          "value": "windows"
+        }
+      }
+    ],
+    "dateRange": {
+      "end": 1594005719000,
+      "start": 1593832919000
+    },
+    "description": "Template for investigating host events",
+    "templateTimelineId": "HostSysProcess", <2>
```

Comment:
Cool this would be useful! but we usually use uuid `e.g.: 6acb2c90-a01c-11ea-8e47-5dc21077d10c` for this field, so I think it would be better to align it.